src/test/java/pages/HomePage.java

```diff
@@ -1,19 +1,63 @@
 package pages;
-
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import java.util.List;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class HomePage extends BasePage {
-
-    By userAvatarIcon = By.cssSelector("img.avatar");
-
-    public HomePage( WebDriver givenDriver) {
+    String playlistName = "my";
+    String newPlaylistName = "lalala";
+    By addNewPlaylist = By.xpath("//*[@title='Create a new playlist']");
+    By newPlaylist = By.xpath("//li[text()='New Playlist']");
+    By nameField = By.xpath("//input[@name='name']");
+    By myPlaylist = By.xpath("//li/a[text()='" + playlistName + "']");
+    By editButton = By.xpath("//li[text()='Edit']");
+    By editField = By.xpath("//input[@name='name']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");